acisfp_check/acis_obs.py

```diff
@@ -101,7 +150,7 @@ def find_obsid_intervals(cmd_states):
     for eachstate in cmd_states:
 
         # Make sure we skip maneuver obsids explicitly
-        if 50000 > eachstate['obsid'] >= 38001:
+        if 60000 > eachstate['obsid'] >= 38001:
```

Comment:
Do we want the 60000s excluded too?

Reply:
No because we need to check the cold ECS measurements as well.